docs/site/next.config.ts

```diff
@@ -40,11 +40,6 @@ const config = {
   },
   redirects() {
     return [
```

Comment:
```suggestion
    return [
      {
        source: "/docs/getting-started",
        destination: "/repo/docs/getting-started/installation",
        permanent: true,
      },
```

If the Getting Started index page is no longer needed, it would be more appropriate to update the redirect rule to point directly to `/getting-started/installation` instead.